pop-api/src/v0/assets/pallets/assets.rs

```diff
@@ -0,0 +1,492 @@
+#![allow(dead_code)]
+
+use crate::{Balance, RuntimeCall, *};
+use ink::prelude::vec::Vec;
+use primitives::{AssetId, MultiAddress};
+use scale::{Compact, Encode};
+
+type Result<T> = core::result::Result<T, PopApiError>;
+
+/// [Pallet Assets](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/assets/src/lib.rs):
+/// 1. Dispatchables
+/// 2. Read state functions
+///
+/// 1. Dispatchables within pallet assets (TrustBackedAssets instance) that can be used via the pop api on Pop Network:
+/// - create
+/// - start_destroy
+/// - destroy_accounts
+/// - destroy_approvals
+/// - finish_destroy
+/// - mint
+/// - burn
+/// - transfer
+/// - transfer_keep_alive
+/// - force_transfer
+/// - freeze
+/// - thaw
+/// - freeze_asset
+/// - thaw_asset
+/// - transfer_ownership
+/// - set_team
+/// - set_metadata
+/// - clear_metadata
+/// - approve_transfer
+/// - cancel_approval
+/// - force_cancel_approval
+/// - transfer_approved
+/// - touch
+/// - refund
+/// - set_min_balance
+/// - touch_other
+/// - refund_other
+/// - block
+
+/// Issue a new class of fungible assets from a public origin.
+pub(crate) fn create(
+	id: AssetId,
+	admin: impl Into<MultiAddress<AccountId, ()>>,
+	min_balance: Balance,
+) -> Result<()> {
+	dispatch(RuntimeCall::Assets(AssetsCall::Create {
```

Comment:
Just a few points to stress (nothing related to your code)
1. Routing through the chain extension is crucial here
2. If pallet-assets::Create changes, the contract will be broken
3. So, we need comprehensive testing to check if any pallets break our API
4. If so, we need to introduce a [shim](https://en.wikipedia.org/wiki/Shim_(computing)) in the chain extension for the affected versions of the Pop API.

We probably want to add this to the API spec.

Reply:
Done